tests/phpunit/elementor/modules/usage/test-module.php

```diff
@@ -439,8 +442,96 @@ public function test_dynamic_control() {
 		self::$document->save( $document );
 
 		// Check element without dynamic control.
-		$global_usage = get_option( Module::OPTION_NAME, [] );
+		$global_usage = get_option( Module::ELEMENTS_OPTION_NAME, [] );
 
 		$this->assertEquals( 0, count( $global_usage[ $doc_name ]['heading']['controls'] ) );
 	}
+
+	/**
+	 * TODO: This test will analyze only `$global_usage` of `wp-post' since in multisite kit is somehow shared,
+	 * And will be added to global usage.
+	 */
+	public function test_page_settings() {
```

Comment:
Can you add 
// Arrange
// Act
// Assert
comment?

and maybe split this test into 3 or 4 tests?

Reply:
Handled.